eng/pipelines/stress-test-release.yml

```diff
@@ -64,14 +64,12 @@ jobs:
     - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
       parameters:
         Repositories:
-          - Name: Azure/azure-sdk-tools
-            Commitish: ${{ parameters.DeployFromBranchOrCommit }}
-            WorkingDirectory: $(System.DefaultWorkingDirectory)/Azure/azure-sdk-tools
           - Name: $(Repository)
             Commitish: ${{ parameters.DeployFromBranchOrCommit }}
             WorkingDirectory: $(System.DefaultWorkingDirectory)/$(Repository)
         Paths:
           - '/tools'
```

Comment:
Do all the repo's have a tools directory? Should this be eng?

Reply:
`/tools` is only here because the stress tests in the `azure-sdk-tools` repo are defined in `/tools`. I wonder if it's better to just add `/*` and do away with sdk/tools?

Reply:
Sparse checkout does work for non-existent paths though.

Reply:
I guess it really depends on what the goal is here. If you just want to avoid the recordings then yes probably include everything except those.

Reply:
You will also likely need to the root files for other languages like .NET as well.

Reply:
Oh you mean in cases where .net stress test packages might need the root proj files to build?

Reply:
Exactly if the are standard .NET project they will need to import the root props.